tinyagentos/routes/agent_auth_requests.py

```
@@ -0,0 +1,302 @@
+            status_code=409,
+            detail="request was decided concurrently; check current status",
+        )
+
+    return {"status": "refused"}
+
+
+@router.get("/api/agents/auth-requests")
+async def list_auth_requests(
+    request: Request,
+    status: Optional[str] = "pending",
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)